vmdb/app/models/ems_refresh/parsers/kubernetes.rb

```diff
@@ -6,6 +6,7 @@ def self.ems_inv_to_hashes(inventory)
 
     def initialize
       @data = {}
+      @data_index = {}
```

Comment:
Shouldn't this already be defined if you are using the process_collection pattern?  Same goes for the `@data` above it.